scripts/make_changelog

```diff
@@ -31,7 +31,7 @@ for i in $(seq 0 $I); do
   LABEL=$(echo $LABELS | jq -r ".[$i]")
   case $LABEL in
   "null")
-      echo "## Unclassified"
+    echo "## Unclassified (Move user-facing items to correct section. Delete the rest)"
```

Comment:
We should also check that the items in the other section are user-facing. We could mention it in either
1. Release checklist 
2. As a HTML comment in the generated changelog